skyllh/core/expectation_maximization.py

```diff
@@ -222,7 +147,7 @@ def em_fit(self, fitparams, n=1, tol=1.e-200, iter_max=500, sob_thresh=0, initia
         """
         ana = self.ana
 
-        ratio = self.get_energy_spatial_signal_over_backround(fitparams)
+        ratio = self.get_energy_spatial_signal_over_background(fitparams)
```

Comment:
The function to get the SoB values should probably be an argument to the constructor of the EM class. Because that function needs knowledge about the LLH function structure.